src/TestSuite/TestCase.php

```diff
@@ -80,6 +80,25 @@ abstract class TestCase extends BaseTestCase
      */
     protected $_configure = [];
 
+    /**
+     * @param string $name Name
+     * @param array $data Data
+     * @param string $dataName Data name
+     */
+    public function __construct($name = null, array $data = [], $dataName = '')
+    {
+        $this->initialize();
```

Comment:
Could we call this method from `getFixtures()` instead?  That method was added in 4.0 to enable defining fixtures outside of the property.

Reply:
My main concern with the getFixtures() method was the side effect of having to call add..() inside.
But I guess that's a fair compromise for now.